### PR TITLE
Fix warnings in user info

### PR DESCRIPTION
**Pull request by antrix1989**

### CODE REVIEW

File: ADAL/src/ADUserInformation+Internal.m [hunk 1]
```diff
@@ -53,18 +33,20 @@ -(NSString*) property \
 return nil; \
 }
 
-@implementation ADUserInformation (Internal)
+NSString *const ID_TOKEN_SUBJECT = @"sub";
```
Comment (oldalton) [inline ADAL/src/ADUserInformation+Internal.m]: those constants are now double :)?

Comment (jasoncoolmax) [reply ADAL/src/ADUserInformation+Internal.m]: @oldalton by "double" you mean it is also in `MSIDIdTokenWrapper`?

Comment (oldalton) [reply ADAL/src/ADUserInformation+Internal.m]: never mind, I didn't notice it was changed from static to extern and I saw two additions :) 
I have a different question now, if they're only used in ADUserInformation.m, why we need to declare them here?

Comment (antrix1989) [reply ADAL/src/ADUserInformation+Internal.m]: It is because we need to share them between internal & public files.

**Pull request merged**